templates/base.html

```diff
@@ -0,0 +1,6 @@
+{{define "base"}}
+<html>
+    <head>{{template "head" .}}</head>
```

Comment:
As I understand it, this should load templates that have been assigned as "head" and "body", and which templates are assigned to these can be swapped around. So for example, in handleHome, I fetch the contents of home.html and place it where "body" is in base.html.

But how do I do this in the handler code?

Reply:
The answer is in the StackOverflow link that we used for inspiration. We also don't need a custom head template, we can pass something like `{{ .Title }}` directly.